regression/cbmc-smt2-z3/struct/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+
+^EXIT=6$
+^SIGNAL=0$
+^VERIFICATION ERROR$
```

Comment:
Oh this is because the assertion doesn't hold.

With a trivial property, CBMC was just simplifying it to `true` and was not even generating SMT queries. I can try to check a different property that won't be trivially simplified to true and would actually require an SMT solver. Or, can I disable all optimizations?

Reply:
I would avoid either one of these options unless they are specifically tested.
If it bugs you that this is a satisfiable instance (which it shouldn't) then you could use s.th. like
`if(t.a>10) assert(t.a > 0);`

Reply:
Thanks everyone! I implemented Daniel's suggestion above. Marking this one as resolved.